src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableInterlocked.cs

```diff
@@ -12,6 +12,48 @@ namespace System.Collections.Immutable
     /// </summary>
     public static class ImmutableInterlocked
     {
+        /// <summary>
```

Comment:
There's nothing about this method specific to immutable collections.  Is this the best place for it?